DualCore/cm4/HelloWorld_cm4.cproject.yml

```diff
@@ -60,12 +47,17 @@ project:
     - group: Application
       files:
         - file: ./hello_world_core0.c
-        - file: ./RTE/Device/K32L3A60VPJ1A_cm4/K32L3A60xxx_cm4_flash.scf
-          for-compiler: AC6
+
     - group: Middleware
       files: 
         - file: ../middleware/multicore/mcmgr/src/mcmgr.c
         - file: ../middleware/multicore/mcmgr/src/mcmgr.h
         - file: ../middleware/multicore/mcmgr/src/mcmgr_internal_core_api.h
         - file: ../middleware/multicore/mcmgr/src/mcmgr_internal_core_api_k32l3a6.c
         - file: ../middleware/multicore/mcmgr/src/mcmgr_mu_internal.c
+
+  linker:
+    - script: ./RTE/Device/K32L3A60VPJ1A_cm4/K32L3A60xxx_cm4_flash.scf
+      for-compiler: AC6
+    - script: ./RTE/Device/K32L3A60VPJ1A_cm4/K32L3A60xxx_cm4_flash.ld
+      for-compiler: GCC
```

Comment:
should we have an empty last line in the files?

Reply:
Sure.